packages/eslint-config/browser.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  extends: [
+    './index.js',
+  ],
+  parser: 'babel-eslint',
+  env: {
+    browser: true,
+  },
+  rules: {
+    'no-console': ['error', { allow: ['warn', 'error'] }],
+
+    // common ones
+    // biblio
+    semi: ['error', 'never'],
+    quotes: [
+      'error',
+      'single',
+      { avoidEscape: true, allowTemplateLiterals: true },
+    ],
+
+    // wf
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+
+    // ydr
+
+    // bib-service-projects
+    'require-atomic-updates': 'warn',
+  },
```

Comment:
The fact that these were originally linked to specific projects still makes them suitable for node or jest environments, I would move them to the shared file 🙏🏽

Reply:
The references to different projects are left there only to indicate that we did a "merge" of all the different preferences everyone had. To highlight that it was done as democratically as possible.

Reply:
After reviewing it again, these rules that were in the index.js file are all good for node.js, including the extended configurations.